grails-app/views/notify/dulFormLink.gsp

```diff
@@ -1,9 +1,13 @@
 <p>
-Hello,
-This email is a request from your collaborator at the Broad Institute to have your IRB or Ethics Committee of record
-complete the linked <b><i>Data Use Letter</i></b> form by interpreting the consent form for this study.<br>
-Could you please share <a href="${details}">this link</a> and/or email with a member of your IRB or Ethics Committee so they can proceed with filling out and submitting the Data Use Letter form?<br>
-If you have any questions, please do not hesitate to email your collaborator (cc'd) or ORSP@broadinstitute.org.<br>
-Thank you,<br>
-Sent on behalf of Broad Institute's Office of Research Subject Protection (ORSP)
+In preparation for deposition to a data repository, the Broad Institute requests that you have your IRB
+or Ethics Committee complete the linked Data Use Limitations form, based on their review of the informed
+consent form used to collect the samples from which data were generated.
+Please share <a href="${details}">this link</a> and/or email with a member of your IRB or Ethics Committee so they can proceed
+with the consent form review and completion of the Data Use Limitations form. If you have any questions,
+please do not hesitate to email your Broad collaborator (cc'd) or the Broad's Office of Research
+Subject Protection at ORSP@broadinstitute.org.
```

Comment:
nit: note that we have both `orsp-portal@broadinstitute.org` and this value here - please confirm with the team, but I think 'orsp-portal@broadinstitute.org' is correct.